src/pages/blog/how-to-thrive-in-an-automated-society.pug

```diff
@@ -0,0 +1,184 @@
+
+            p To truly get the most from automation, you need to look at entire processes, and intelligently apply automation where it can have the greatest effect. From here, you repeatedly working on automating other parts of the process to eliminate bottlenecks and make the entire operation efficient. We wrote <a href="https://weareleaf.com/blog/automate-where-it-matters-with-process-maps" rel="noopener" target="_blank">a blog post on using process maps to achieve this</a>.
+
+            h4 Invest in your own software
+
+            p Many businesses rely too heavily on staff to manually facilitate interactions between separate pieces of off the shelf automation software; copying data, manually creating reports, and updating the same data across many software products is common.
+
+            p By investing in your own bespoke automation software to connect these systems, you can fully automate much of your businesses operation, and create unique intellctual property in the process. A common example of this is integrating the data from your CRM product, your web portal, and your analytics product together.
+
+            p Integrating off the shelf automation products with one another improves scalability, improves data quality, reduces data duplication, and even makes your business easily programmable for new internal products, or other B2B arrangements via an API.
+
+            h4 Niche and scale
+            p Traditional logic tells us that in order to thrive, should remain generalists and attract as much business as possible. Unfortunately, it's hard to grow a business that caters to everyone's needs an automated way, and if you do succeed, you end up blending into a crowd of countless competitors that look the same.
+
+            p Picking a niche makes it easier to automate, as your customers increasingly fit a similar mold that you can build robust processes around. You can then scale this while standing out from the crowd as the clear choice for your particular niche.
+
+            p At a global scale, you can build a decent sized business around even relatively obscure niches, and if you want to grow even further, you can begin to cater for a broader range of customers from a position of notoriety and strength.
+
+            h4 Embrace new revenue streams
+            p As you automate, you'll free up time for your business to work on other things. You can compound the benefit you get from automation by re-using existing automated processes to cater for new markets, or building new products entirely. Common examples include:
+
+            ul
+              li Charging other organisations to integrate with your services on a self-service basis.
+              li Becoming established in a niche at a global level, and <em>then</em> beginning to generalise.
+              li Building entirely new products that complement your existing product offerings.
+              li Teaching, and consulting for other businesses.
```

Comment:
I feel like we either don't need the comma (because they're linked), or they should be separate points (because they're not linked).